en/components/slider.md

```diff
@@ -6,18 +6,18 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
 
 # Slider
 
-Use the Slider Component to allow the user to select a single value or specify a range of values by choosing values for both its ends. The Slider is visually identical to the [Ignite UI for Angular Slider Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/slider.html)
+Use the Slider Component to allow the user to select a single value or specify a range of values by choosing values for both its ends. The Slider is visually identical to the [Ignite UI for Angular Slider Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/slider/slider.html)
 
 ## Slider Demo
 
 <img class="responsive-img" src="../images/slider_demo.png" srcset="../images/slider_demo@2x.png 2x" />
 
 ## Type
 
-The Slider offers a variant with one thumb for selecting a single value and with two thumbs for specifying a range.
+The Slider allows to choose from two types - continuous and discrete. They each offer a variant with one thumb for selecting a single value and with two thumbs for specifying a range.
```

Comment:
Take out the one and two thumbs and place them in a separate sub-section. 
Move the state section for enabled and disabled after it and then continue with theme and styling.